### PR TITLE
Consistency updates for street light placement

### PR DESCRIPTION
**Pull request by chaosvolt**
## Summary

SUMMARY: Balance "Place street lights in areas previously updated to be more consistent"

## Purpose of change

Some follow-up updates to https://github.com/cataclysmbnteam/Cataclysm-BN/pull/3265 because since then I've gone and set an actual precedent for where the street lights should be placed relative to the street and some of those locations I initially updated kinda don't follow that new convention. XD

## Describe the solution

1. Added a streetlight on the right side to malls at the road connection. Kept the parking lot lights unchanged since is a big location.
2. Moved the street lights in banks over to the right side.
3. Set public space to have just one light over on the right.
4. Shifted placement of street lights in home improvement stores.
5. Updated parking lot placement of street lights in hospitals.
6. Adjusted placement of street light in those janky fucking megastore parking lots.
7. Added street light to motels in the new standard spots, as with malls left the parking lot lights intact since why not.
8. Added street lights in the expected road connections to football stadiums, leaving the original lights in place for effect.
9. Updated light placement at town hall.
10. Misc: Fixed football stadium not having a pole on the opposite end of the field.

Left abandoned shopping plaza unchanged as the edge is occupied wholly by the chain-link fence. Bus station placement also seemed close enough that it'd still look right.

## Describe alternatives you've considered

Screaming.

## Testing

1. Checked affected files for syntax and lint errors.
2. Load-tested to make sure I didn't miss an undefined tile.

## Additional context

At some point football stadiums need some tweaks so that it'll be possible to grant them lighting. main thing is the lines of grass are oddly placed and a bit uneven.

### CODE REVIEW

**Comment by olanti-p** [comment]
NoHope mapgen failure
https://github.com/cataclysmbnteam/Cataclysm-BN/actions/runs/6422683075/job/17439695941?pr=3369#step:11:1963

**Review by scarf005 — COMMENTED** [review]
https://github.com/cataclysmbnteam/Cataclysm-BN/pull/3348#issuecomment-1751651433

could you also check gym mapgen?

File: data/json/mapgen/gym.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
         " ": "t_floor",
         "+": "t_door_c",
         ",": "t_sidewalk",
-        "'": "t_sidewalk",
+        "'": "t_dirt",
```
Comment (scarf005) [inline data/json/mapgen/gym.json:39]:
```
 ~/r/c/Cataclysm   street-light-followup-revisit $  ./cata_test-tiles --user-dir=all_modded --mods=bn,crazy_cataclysm,desertpack,limit_fungal_growth,cbm_slots,magiclysm,ruralbiome,smart_house_remote_mod,classic_zombies,aftershock,DinoMod,Only_Wildlife,test_data,No_Ferals,UDP_BN_FAKE_SNOW,stats_through_kills,Old_Mutations,StatsThroughSkills,no_hope,speedydex,MMA,elevated_bridges,No_Rail_Stations,disable_lifting,more_classes_scenarios,alt_map_key,saveload_lua_test,no_npc_food,fuji_mpp,No_Blobs,my_sweet_cataclysm '~*' -d yes --use-colour yes --rng-seed time
Randomness seeded to: 1696694419
01:00:19.087 INFO : Randomness seeded to: 1696694419
01:00:19.088 INFO : C locale on startup: 'ko_KR.UTF-8'
01:00:19.088 INFO : C++ locale on startup: 'ko_KR.UTF-8'
01:00:19.089 WARNING : Failed to detect system UI language.
01:00:19.091 INFO : Number of render drivers on your system: 3
01:00:19.091 INFO : Render driver: 0/opengl
01:00:19.091 INFO : Render driver: 1/opengles2
01:00:19.091 INFO : Render driver: 2/software
01:00:19.091 INFO : Set debug classes to: [DEBUGMSG MAIN LUA]
01:00:19.120 INFO LUA : SHR: preload.
01:00:19.120 INFO LUA : SLT: preload
Terminated: Json error: data/json/mapgen/gym.json:170:13: duplicate entry in json object

        "'": "t_sidewalk",
        ".": [ "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt" ],
        "'":
            ^
             "t_dirt",
        "5": "t_wall_glass",
        "=": "t_door_glass_c",

Make sure that you're in the correct working directory and your data isn't corrupted.
```

i think this and line 51 is causing conflict.

**Review by scarf005 — CHANGES_REQUESTED** [review]

**Comment by chaosvolt** [comment]
Whoops, just woke up and overlooked that D:

**Review by olanti-p — APPROVED** [review]

**Review by scarf005 — APPROVED** [review]

**Pull request merged**